### PR TITLE
R - package writing workshop tutorial with link to the pdf

### PR DESCRIPTION
**Pull request by siddhantshandilya**

### CODE REVIEW

File: r-package-writing.Rmd [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "R Package Writing Workshop"
+author: "Mohit Chander Gulla, Siddhant Shandilya"
+date: "10/25/2019"
+output: html_document
+---
```
Comment (bojum) [inline r-package-writing.Rmd]: Remove this part

**Review by bojum — CHANGES_REQUESTED** [review]
- Remove the yml header.
- Put your workshop name at the top (see [here](https://jtr13.github.io/cc19/index.html#preparing-your-.rmd-file))

**Comment by siddhantshandilya** [comment]
Made the required changes. Please merge! Thank you

**Comment by bojum** [comment]
🎉 Merged! Thanks for your contribution!

**Pull request merged**